test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -448,11 +467,44 @@ class test_uct_cm_sockaddr : public uct_test {
          self->m_server_connect_cb_cnt++;
     }
 
+    int get_ep_index (uct_ep_h ep) {
+        int i;
+
+        for (i = 0; i < (2 * m_enhance_clients_num); i++) {
+            if (m_all_eps[i].ep == ep) {
+                return i;
+            }
+        }
+
+        return -1;
+    }
+
+    void common_enhance_test_disconnect(uct_ep_h ep) {
+        int index;
+
+        index = get_ep_index(ep);
+        EXPECT_GT(index, -1);
```

Comment:
better check against valid values then magic value:
`EXPECT_GE(index, 0);`

Reply:
it's not a magic value, -1 is a valid return value from the function.
but ok, i'll change..